### ISSUE TITLE
Anti-IA2A

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7241]
**Preferred term label:**
Anti-IA2 antibody positivity

**Synonyms**
Anti-tyrosine phosphatase IA-2 antibody positivity, Anti-IA2A, Anti-islet antigen-2 antibody positivity, ICA512

**Definition (free text, please give PubMed ID)**
PMID: 9693976

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:103919

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by MaudDLVT** [comment]
To merge with HP:5000094

**Comment by pnrobinson** [comment]
Adding with primary label Anti-islet antigen-2 antibody positivity
There is no HP:5000094
